CodeReview/Amie_code.py

```diff
@@ -9,9 +9,12 @@
 %matplotlib inline
 #Load data
 data = pd.read_csv('../Data/2014_PortTownsend.csv', parse_dates=['Date Time'])
+
 import plotly.plotly as py
 import plotly.graph_objs as go
+#Use date time index
 data_datetime = pd.DatetimeIndex(data['Date Time'])
+#Edit data adding new columns
 def f(data):
     data = data.copy()
```

Comment:
You should just be able to use the original dataframe.  I don't know why you would need to copy it.  Also, if you wanted to keep it this way, you should use a different variable name.  e.g.  $ tides = data.copy()